src/main/java/org/jboss/pnc/deliverablesanalyzer/model/FinderResultCreator.java

```diff
@@ -60,6 +64,50 @@ public static FinderResult createFinderResult(String id, URL url, Map<BuildSyste
                 .build();
     }
 
+    private static void setLicenseInformation(Artifact.ArtifactBuilder builder, KojiLocalArchive localArchive) {
+        Set<LicenseInfo> licenses = Optional.ofNullable(localArchive.getLicenses())
+                .orElse(Collections.emptySet())
+                .stream()
+                .map((org.jboss.pnc.build.finder.core.LicenseInfo license) -> {
+                    LicenseInfo.LicenseInfoBuilder licenseBuilder = LicenseInfo.builder()
+                            .comments(license.getComments())
+                            .distribution(license.getDistribution())
+                            .name(license.getName())
+                            .spdxLicenseId(license.getSpdxLicenseId())
+                            .url(license.getUrl());
+
+                    org.jboss.pnc.build.finder.core.LicenseSource source = license.getSource();
+                    if (source != null) {
```

Comment:
I prefer guard ifs instead of double negative else (if source is not not null)

Reply:
Actually, `source` should never be `null`, so you don't need this check. The default is to set to `UNKNOWN`, but build finder will always fill it with some other non-null value anyway.